test/bun.js/child_process.test.ts

```diff
@@ -9,12 +9,11 @@ import {
   execFileSync,
   execSync,
 } from "node:child_process";
+import { tmpdir } from "node:os";
 
 const debug = process.env.DEBUG ? console.log : () => {};
 
-const platformTmpDir = `${process.platform === "darwin" ? "/private" : ""}${
```

Comment:
@ThatOneBro `process.env.TMPDIR` is macOS-specific

Reply:
Ah. I'll start testing on my linux system too, I thought it was there in Linux too